docs/api.md

```diff
@@ -729,19 +728,26 @@ Shortcut for [page.mainFrame().evaluate(pageFunction, ...args)](#frameevaluatepa
 #### page.evaluateHandle(pageFunction, ...args)
 - `pageFunction` <[function]|[string]> Function to be evaluated in the page context
 - `...args` <...[Serializable]|[JSHandle]> Arguments to pass to `pageFunction`
-- returns: <[Promise]<[JSHandle]>> Resolves to the return value of `pageFunction`
+- returns: <[Promise]<[JSHandle]>> Promise which resolves to the return value of `pageFunction` as in-page object (JSHandle)
```

Comment:
There are many `evaluateHandle` calls: `page.evaluateHandle`, `frame.evaluateHandle` and `elementHandle.evaluateHandle`. We try to keep documentation consistent between them all; so all the changes has to be mirrored there as well.

Reply:
Cool, I will update there as well.